test-suite/coq-makefile/quick2vo/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+a=`uname`
+
+. ../template/init.sh
+
+coq_makefile -f _CoqProject -o Makefile
+# vio2vo is broken on Windows (#6720)
+if [ "$a" = "Darwin" -o "$a" = "Linux" ]; then
```

Comment:
You should print a warning if the test is not run.

Reply:
I copied this from `misc/deps-utf8` which also does not. How should the warning look like?

Reply:
OK forget about this, there are many tests that are not run without printing a warning and this should be addressed globally.